README.app.md

```diff
@@ -40,9 +40,6 @@ Run test
 
 Tests files should be in the `tests/` directory and under the namespace `Tests\\`
 
-* *Supervisor*: http://{{ app_host }}.dev:9001
```

Comment:
Fortunately, yes :)
The fact is the role is still not ready for jessie. In few days, i will add it back again.